Source/WebCore/layout/integration/grid/LayoutIntegrationGridLayout.cpp

```diff
@@ -41,7 +45,25 @@ GridLayout::GridLayout(RenderGrid& renderGrid)
 
 void GridLayout::layout()
 {
-    // FIXME implement this
+    // FIXME: implement this.
+}
+
+TextStream& operator<<(TextStream& stream, const GridLayout& layout)
+{
+    stream << "GridLayout@" << &layout;
+    stream << " gridBox=" << &layout.gridBox();
+    size_t index = 0;
+    for (CheckedRef<const Layout::Box> box : Layout::formattingContextBoxes(layout.gridBox())) {
+        stream << "\n  [" << index++ << "] box=" << static_cast<const void*>(box.ptr());
+        stream << " anonymous=" << (box->isAnonymous() ? "yes" : "no");
+        stream << " establishesContext=" << (box->establishesFormattingContext() ? "yes" : "no");
+        stream << " display=" << static_cast<unsigned>(box->style().display());
```

Comment:
Seems like we have a text stream operator for `DisplayType` in `RenderStyleConstants.h` so could this just be 
```
 stream << " display=" << box->style().display();
```